### ISSUE TITLE
API Key Environment Variable doesn't behave as expected in Python

### ISSUE THREAD

**Issue by kmanwar89** [meraki/dashboard-api-python#150]
This is my first time interacting with this particular library, and I'm still learning about API's, so please pardon my ignorance.

If I use Postman, I'm able to interact with the Dashboard API without issues. When using Python, the environment variables don't work as expected.

**Development Environment**
- Dell Inspiron Laptop
- Ubuntu 20.04 - fresh install
- Python 3.8
- VSCode
- Postman
- bash for shell, using the default terminal with Ubuntu

**Expected behavior**
- Exporting an environment variable within the terminal, and validating it from the output of the *env* command, should result in the variable being passed through to the instantiated object via the os.environs.get() method, which is present in __init__.py as witnessed here: 

![Screenshot from 2021-03-11 12-54-21](https://user-images.githubusercontent.com/11323060/110832108-f09d9180-8268-11eb-99e9-3ea23febf11c.png)

**Actual behavior**
- Environment variable is not passed into the object, resulting in a 401 'Invalid API Key' error
- This behavior is present whether through the IDE (VSCode), executed from a terminal or directly through the interactive Python interpreter.

**Steps to fix**
Use one of several methods to manage the API token
- Export the variable in the shell's *.bashrc* file and use *os.environ.get(<variable name>)*. --- **This shouldn't be required as the code is already present in the __init__, so this is duplicate effort**
- Leverage the *python-dotenv* library. This involves creating a *.env* file in the same directory as the code, adding it to .gitignore and adjusting file permissions accordingly to secure access to only the user. -- **Secure, but inconvenient**
- Manually defining the API token into the code as plaintext -- **least secure, most convenient**
- Write a separate bash script to handle managing the API token -- **requires additional bash scripting knowledge above and beyond Python and shouldn't be necessary**

**Requested response from developers**
- Can the documentation be updated to reflect the most preferred method, or clarify that additional steps are needed to manage the environment variable? The current documentation is sparse and simply assumes that exporting will work without any further steps.

**Comment by TKIPisalegacycipher** [comment]
I'm not able to reproduce this on Ubuntu or Windows. Do you have any other
steps to reproduce?

**Comment by kmanwar89** [comment]
Hi there,

I'm not entirely sure what happened, but while trying to reproduce this for a friend today at work, I couldn't re-create the behavior. I spun  up a fresh Ubuntu VM to see if I can reproduce it, but I can't. I think what may have happened is the environment variable wasn't being read into VSCode until I restarted VSCode. I may have also re-*sourced* my *.bashrc* file, which probably made that variable "active" within the environment, allowing VSCode to then retrieve it and pass it along into the API call.

If this happens again, I will be sure to detail exactly what I did to produce the error. Thank you.
